<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegTikTokChrome</class>
 <widget class="QMainWindow" name="RegTikTokChrome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegTikTokChrome</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnStart">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Bắt đầu</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/icons8-play-100.png</normaloff>:/images/icons8-play-100.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Dừng</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/icons8-stop-100.png</normaloff>:/images/icons8-stop-100.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Luồng:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="threadCount">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="versionChrome">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>75</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFolderAvatar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>FolderAvatar</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pathAvatar">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFolderChrome">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>FolderPortable</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pathChrome">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFileGmail">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>File Gmail</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/images/icons8-txt-96.png</normaloff>:/images/icons8-txt-96.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pathGmail">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Omocaptcha.com</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="keyOmocaptcha"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>gmail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>passMail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>userName</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>cookie</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>status</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
